node/tests/redis-modules/RedisModules.test.ts

```diff
@@ -0,0 +1,348 @@
+import { afterAll, beforeAll, describe, expect, it } from "@jest/globals";
+import { v4 as uuidv4 } from "uuid";
+import {
+    BaseClientConfiguration,
+    ConditionalChange,
+    InfoOptions,
+    ProtocolVersion,
+    RedisClient,
+    RedisClusterClient,
+    parseInfoResponse,
+} from "../../";
+import * as json from "../../src/redis-modules/Json";
+import { RedisCluster, getFirstResult } from "../TestUtilities";
+
+type Context = {
+    client: RedisClusterClient | RedisClient;
+};
+
+type BaseClient = RedisClusterClient | RedisClient;
+
+const TIMEOUT = 20000;
+
+describe("RedisModules", () => {
+    let testsFailed = 0;
+    let cluster: RedisCluster;
+    let loadModuleList: string[];
+
+    beforeAll(async () => {
+        const args = process.argv.slice(2);
+        const loadModuleArgs = args.filter((arg) =>
+            arg.startsWith("--load-module="),
+        );
+        loadModuleList = loadModuleArgs.map((arg) => arg.split("=")[1]);
```

Comment:
You're assuming --load-module was passed, can you check it and fail the tests with appropriate message if it doesnt?